sentinelsat/sentinel.py

```diff
@@ -79,10 +79,25 @@ def __init__(
         self.session.headers["User-Agent"] = self.user_agent
         self.show_progressbars = show_progressbars
         self.timeout = timeout
+        self._dhus_version = None
         # For unit tests
         self._last_query = None
         self._last_response = None
 
+    @staticmethod
+    def _api2dhus_url(api_url):
+        return api_url.replace("apihub", "dhus")
+
+    @property
+    def dhus_version(self):
+        if self._dhus_version is None:
+            dhus_url = self._api2dhus_url(self.api_url)
```

Comment:
Since there is only 1 "apihub" and multiple "dhus" I think it would be nicer to have this in a `try...except` block, i.e. try to get the version first and then call ` _api2dhus_url`. 

I know in reality it does not make much of a difference but from a code clarity standpoint I think it increases the readability if we treat the exception to the rule as an actual `exception`.

Reply:
Ignore the Python 3.4 build error for now.It is a dependency issue that we need to resolve.

Reply:
> Since there is only 1 "apihub" and multiple "dhus" I think it would be nicer to have this in a `try...except` block, i.e. try to get the version first and then call ` _api2dhus_url`.

I wrapped the call in a `try...except` block. Although, in my opinion the code is now more verbose and less readable. But it is also not a big deal. I'm fine with both versions.